network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/annotated_image_block.html

```diff
@@ -1,13 +1,30 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% image value.image original as img %}
-<figure class="my-default">
-  <img src="{{ img.url }}" alt="{{ value.altText }}" class="w-100" />
-  {% if value.caption %}
-  <figcaption class="body-small d-block mt-2">
-    {% if value.captionURL %}<a href="{{ value.captionURL }}">{% endif %}
-    {{ value.caption }}
-    {% if value.captionURL %}</a>{% endif %}
-  </figcaption>
-  {% endif %}
-</figure>
+
+{% if value.wide_image %}
+    </div> <!-- End .blog-body -->
+  </div> <!-- End .cms -->
+</div> <!-- End .col-lg-8 -->
+
+<div class="col-lg-12">
+{% endif %}
+
+  <figure class="my-default">
+    <img src="{{ img.url }}" alt="{{ value.altText }}" class="w-100" />
+    {% if value.caption %}
+    <figcaption class="body-small d-block mt-2">
+      {% if value.captionURL %}<a href="{{ value.captionURL }}">{% endif %}
+      {{ value.caption }}
+      {% if value.captionURL %}</a>{% endif %}
+    </figcaption>
+    {% endif %}
+  </figure>
+
+{% if value.wide_image %}
+</div>
+
+<div class="py-4 py-md-5 col-lg-8 offset-lg-2">
```

Comment:
Yes there is. Because its a stream block it could be used anywhere in the future. The ideal scenario would be using 2 annotated image blocks for both publications and blog (one each), and keep that pattern in the future for future use cases. It creates less DRY code, but keeps it flexible because the blog vs publication templates were setup differently. And a future page where we need to break out of the normal grid would likely be different yet again.